<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
<<<<<<< HEAD
    <width>766</width>
=======
    <width>782</width>
>>>>>>> origin/master
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ADN - Plot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
<<<<<<< HEAD
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>17</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>17</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>67</width>
          <height>17</height>
         </size>
        </property>
        <property name="text">
         <string>Puerto:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboPorts">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>25</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>No hay dispositivos conectados</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushConectar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Conectar</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
=======
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>67</width>
            <height>17</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>67</width>
            <height>17</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>67</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>Puerto:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboPorts">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>25</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>No hay dispositivos conectados</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushConectar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Conectar</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>23</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>60</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Mostrar:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_frecuencia">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Frecuencia</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_capacidad">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Capacidad</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>468</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
>>>>>>> origin/master
      </item>
     </layout>
    </item>
    <item>
<<<<<<< HEAD
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Gráfica 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="PlotWidget" name="graphicsView"/>
        </item>
       </layout>
      </widget>
=======
     <widget class="PlotWidget" name="plot_1">
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>360</height>
       </size>
      </property>
>>>>>>> origin/master
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
<<<<<<< HEAD
     <width>766</width>
=======
     <width>782</width>
>>>>>>> origin/master
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir_datos"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
<<<<<<< HEAD
   <widget class="QMenu" name="menuConfiguraci_n">
    <property name="title">
     <string>Configuración</string>
    </property>
    <addaction name="actionGr_ficas_2"/>
    <addaction name="actionPuertos"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuConfiguraci_n"/>
=======
   <addaction name="menuArchivo"/>
>>>>>>> origin/master
   <addaction name="menuAyuda"/>
  </widget>
  <action name="actionConfiguraci_n">
   <property name="text">
    <string>Configuracion</string>
   </property>
  </action>
  <action name="actionNinguno">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ninguno</string>
   </property>
  </action>
  <action name="actionSearchPorts">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Buscar puertos</string>
   </property>
  </action>
  <action name="actionConectar_dispositivo">
   <property name="text">
    <string>Conectar dispositivo</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Acerca de ADN-Plot</string>
   </property>
  </action>
  <action name="actionContenido">
   <property name="icon">
    <iconset>
     <normaloff>Assets/Help_mark_query_question_support-512.png</normaloff>Assets/Help_mark_query_question_support-512.png</iconset>
   </property>
   <property name="text">
    <string>Contenido</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset>
     <normaloff>Assets/cross_close_quit-128.png</normaloff>Assets/cross_close_quit-128.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionNueva_ventana">
   <property name="text">
    <string>Nueva ventana</string>
   </property>
  </action>
  <action name="actionAbrir_datos">
   <property name="text">
    <string>Abrir datos</string>
   </property>
  </action>
<<<<<<< HEAD
  <action name="actionGr_fica_1">
   <property name="text">
    <string>Gráfica 1</string>
   </property>
  </action>
  <action name="actionPuertos">
   <property name="text">
    <string>Puertos</string>
   </property>
  </action>
  <action name="actionGr_ficas_2">
   <property name="text">
    <string>Gráficas</string>
   </property>
  </action>
=======
>>>>>>> origin/master
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
